ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/site/js/polyfills.js

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright 2019 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+/**
+ * Element.matches()
+ * https://developer.mozilla.org/enUS/docs/Web/API/Element/matches#Polyfill
+ */
```

Comment:
add dependency, don't copy

Reply:
removed this file and added sites library as dependency as there is no changes in both libraries.